client/src/components/common/applicant-info-2.vue

```diff
@@ -177,7 +236,11 @@ export default class ApplicantInfo2 extends NameRequestMixin {
     v => (!v || v.length <= 75) || 'Cannot exceed 75 characters'
 
   ]
-  phoneFaxRules = [
+  phoneRules = [
+    v => !!v || 'Required field',
+    v => (v.length === 0 || v.length === 14) || 'Not a valid Phone number'
```

Comment:
The length for 14 initially confused me. Please add a comment explaining how we get to 14.

Reply:
I referred business-create-ui for this validation

Reply:
Maybe business-create-ui also need a comment :)

Please add a comment here explaining why "14".

Reply:
sure :)